src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -138,6 +140,41 @@ double JacobianPerformanceTest(ModuleType& module,
   return arma::max(arma::max(arma::abs(centralDifference - delta)));
 }
 
+// Simple numerical gradient checker.
```

Comment:
By perturbing the parameters right. I was thinking to reuse the same function for the ``SparseAutoencoderFunctionGradient`` test case, would save some code.